zeek

```diff
@@ -120,11 +137,6 @@ main() {
 	start)
 		#Command(s) needed to start the service right now
 
-		if [ "$running" == "true" ]; then
-			echo "Zeek is already running." >&2
-			exit 0
-		fi
```

Comment:
This block should stay. It's the one under the `readpcap` part that isn't needed.

Reply:
Oops, sorry!  Placed it in the right place now.

Reply:
Confirmed that with this check in the right place in the script I'm able to have Zeek running in the background while readpcap'ing in the foreground.